menu_position.routing.yml

```diff
@@ -37,3 +37,9 @@ menu_position.settings:
     _title: 'Settings'
   requirements:
     _permission: 'administer site configuration'
+
+#Define homepage as default route.
```

Comment:
This should have a space between the `#` and D but I'm not sure this comment is needed.